docs/commands/super.md

```diff
@@ -833,19 +853,19 @@ FROM 'gha.parquet'
 ```
 as
 ```sql
-SELECT rec.login as assignee FROM (
-    SELECT unnest(payload.pull_request.assignees) rec
+SELECT object.login as assignee FROM (
+    SELECT unnest(payload.pull_request.assignees) object
     FROM 'gha.parquet'
 )
 ```
 and for ClickHouse, we had to use `arrayJoin` instead of `unnest`.
 
-SuperSQL's data model does not require these sorts of gymnastics as
+SuperSQL's data model does not require these kinds of gymnastics as
```

Comment:
The word "sort" appears so often for technical reasons, I figured I'd opt for a synonym.